xbmc/guilib/guiinfo/GUIInfoHelper.cpp

```diff
@@ -175,6 +176,26 @@ CGUIListItemPtr GetCurrentListItem(int contextWindow, int containerId /* = 0 */,
   return item;
 }
 
+std::string GetFileInfoLabelFromItem(int info, const CFileItem *item)
```

Comment:
Why pass CFileItem by pointer instead of passing it by reference?

Reply:
Because all of the guiinfo API functions are using raw pointers. So, reason could be consistency. But now hard feelings about that. Do you want me to change this?